### ISSUE TITLE
Refreshing on other pages other than inbox with the compose open does not fire the DOM observers- compose event

### ISSUE THREAD

**Issue by saketh018** [KartikTalwar/gmail.js#720]
Refreshing on other pages other than inbox with the compose open does not fire DOM observers- compose.

Steps to reproduce-
1. open the compose window in pages other than inbox(ex- sent, drafts, etc.)
2.refresh the page 

After the refresh the page we see that that the DOM observers- compose does not get fired.

**Comment by josteink** [comment]
I'm on latest version and I'm not able to reproduce this. Is this still an issue?

**Comment by saketh018** [comment]
Yes, I'm currently on version 1.1.6 and this issue still persists.

**Comment by danilrafiqi** [comment]
i got the same issue

**Comment by akshaycj** [comment]
+1

**Comment by josteink** [comment]
I guess if this is an issue for people, those who encounter it will have to do one of two things:

* manually check for open compose-instances upon extension load
* submit a PR to make gmail.js automatically do the same upon load.

Not much I can do on my end.
